src/connect/components/file-item/file-item.stories.tsx

```diff
@@ -1,24 +1,14 @@
+import {
+    mockNodeOptions,
+    mockUiFileNode,
+    mockUiFolderNode,
+} from '@/connect/utils';
 import { Meta, StoryObj } from '@storybook/react';
 import { FileItem } from './file-item';
 
 const meta: Meta<typeof FileItem> = {
     title: 'Connect/Components/FileItem',
     component: FileItem,
-    argTypes: {
```

Comment:
v 8 of storybook is moving away from explicit arg types. I've had to remove them in some places because they cause typescript errors now.